docs/courses/intro_to_gui.rst

```diff
@@ -0,0 +1,263 @@
+.. _intro_to_gui:
+
+Introduction to the GUI
+=======================
+
+
+Physical System
+---------------
+An electrolyte-containing vesicle with a lipid bilayer membrane that has no ion channels.
+
+Conceptual Model
+~~~~~~~~~~~~~~~~
+
+A capacitor
+
+.. image:: img/bilayer.gif
+    :align: center
+
+
+Simulation
+~~~~~~~~~~
+
+Computational implementation of the conceptual model
+++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+We're going to use the CellBuilder to make a cell that has surface area = 100 um2 and no ion channels.
+
+Alternatively, we could do this in HOC
+
+.. code:: 
+    hoc
+
+    create soma
+    . . . plus more code . . .
+
+or in Python
+
+.. code::
+    python
+
+    from neuron import h
+
+    soma = h.Section(name = 'soma')
```

Comment:
It was like this in the original, but... can you please change the soma line to:

```python
soma = h.Section(name='soma')
```

i.e., No spaces inside the parentheses. It's not the style guide that I would invent, but it is what PEP8 says.